Sources/Widgets/Widgets3D/ResliceCursorWidget/test/testMultipleRotationPlanes.js

```diff
@@ -0,0 +1,365 @@
+              const activeViewName = widget
+                .getWidgetState()
+                .getActiveViewName();
+              const currentViewName = getViewPlaneNameFromViewType(viewType);
+              const keepFocalPointPosition =
+                activeViewName !== currentViewName && canUpdateFocalPoint;
+              updateReslice({
+                viewType,
+                reslice,
+                actor: obj.resliceActor,
+                renderer: obj.renderer,
+                resetFocalPoint: false,
+                keepFocalPointPosition,
+                computeFocalPointOffset,
+                resetViewUp: false,
+              });
+            }
+          );
+        });
+
+        updateReslice({
+          viewType,
+          reslice,
+          actor: obj.resliceActor,
+          renderer: obj.renderer,
+          resetFocalPoint: true, // At first initilization, center the focal point to the image center
+          keepFocalPointPosition: false, // Don't update the focal point as we already set it to the center of the image
+          computeFocalPointOffset: true, // Allow to compute the current offset between display reslice center and display focal point
+          resetViewUp: true, // Need to be reset the first time the widget is initialized. Then, can be set to false, so that the camera view up will follow the camera
+        });
+        obj.renderWindow.render();
+      });
+
+      // ----------------------------------------------------------------------
+      // Defines methods used for testing
+
+      /**
+       * Update all views after being moved axis or center
+       */
+      function updateViews(keepFocalPointPosition = false) {
+        viewAttributes.forEach((obj, i) => {
+          updateReslice({
+            viewType: viewTypes[i],
+            reslice: obj.reslice,
+            actor: obj.resliceActor,
+            renderer: obj.renderer,
+            resetFocalPoint: false,
+            keepFocalPointPosition,
+            computeFocalPointOffset: false,
+            resetViewUp: false,
+          });
+          obj.renderWindow.render();
+        });
+      }
+
+      /**
+       * Update the reslice view and compare it to ground truthes values
+       * @param {ViewType} viewType Defines the viewType
+       * @param {Object} expectedValues Contains ground truthes values that will be compared (focalPoint, viewUp, origin, point1, point2)
+       */
+      function updateAndCompareView(viewType, expectedValues) {
+        const viewObj = viewAttributes.find((obj) => obj.viewType === viewType);
+        const out = updateReslice({
+          viewType,
+          reslice: viewObj.reslice,
+          actor: viewObj.resliceActor,
+          renderer: viewObj.renderer,
+          resetFocalPoint: false,
+          keepFocalPointPosition: true,
+          computeFocalpointOffset: false,
+          resetViewUp: false,
+        });
+        const tolerance = 1e-4;
```

Comment:
declare at the top of the file and uppercase  

Optionally rename to EPSILON (check the rest of the VTK.JS code base to find consistency)